### ISSUE TITLE
IOS 16 is released ... any chance of an update ?

### ISSUE THREAD

**Issue by Lazrhog** [iGhibli/iOS-DeviceSupport#190]

**Comment by kot-ezhva** [comment]
https://github.com/iGhibli/iOS-DeviceSupport/pull/192

**Comment by NSDeveloper** [comment]
You may download released version from #191

**Comment by Lazrhog** [comment]
Yeah I tried that one, but this trick doesn't seem to work anymore.  I downloaded the latest Xcode and extracted the IOS16 folder myself and copied it over, but it still fails to prepare my device for development.  I have Xcode 12.4 and Catalina, this is with iPhone 13 pro max running IOS16.   I would be interested to see if anyone has made this combination work anymore.   It worked fine with IOS15 versions using the folders from this repo

**Comment by kot-ezhva** [comment]
Н

> Yeah I tried that one, but this trick doesn't seem to work anymore. I downloaded the latest Xcode and extracted the IOS16 folder myself and copied it over, but it still fails to prepare my device for development. I have Xcode 12.4 and Catalina, this is with iPhone 13 pro max running IOS16. I would be interested to see if anyone has made this combination work anymore. It worked fine with IOS15 versions using the folders from this repo

Try to use iOS 16.1 - https://github.com/iGhibli/iOS-DeviceSupport/pull/192

**Comment by Lazrhog** [comment]
Thanks for the help.  Not managed to get this working yet.  I'll post back if I do ...

**Comment by kot-ezhva** [comment]
It's work for me 🤷‍♂️

Xcode Version 13.4.1 (13F100)
MacOS 12.3.1 (21E258)

Path: `/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/DeviceSupport/`

**Comment by Lazrhog** [comment]
I'm on xcode 12.4 and Catalina
